packages/backend/src/models/UserModel.ts

```diff
@@ -1,31 +1,202 @@
 import bcrypt from 'bcrypt';
-import { CreateOrganizationUser } from 'common';
+import {
+    CreateInitialUserArgs,
+    CreateOrganizationUser,
+    LightdashUser,
+    SessionUser,
+    UpdateUserArgs,
+} from 'common';
 import { Knex } from 'knex';
-import { createEmail, EmailTableName } from '../database/entities/emails';
+import {
+    createEmail,
+    deleteEmail,
+    EmailTableName,
+} from '../database/entities/emails';
 import { InviteLinkTableName } from '../database/entities/inviteLinks';
 import { createOrganizationMembership } from '../database/entities/organizationMemberships';
+import { createOrganization } from '../database/entities/organizations';
 import { createPasswordLogin } from '../database/entities/passwordLogins';
 import {
-    createUser,
-    DbUserDetails,
-    getUserDetailsByUuid,
+    DbUser,
+    DbUserIn,
+    DbUserUpdate,
     UserTableName,
 } from '../database/entities/users';
-import { NotExistsError, ParameterError } from '../errors';
+import { NotExistsError, NotFoundError, ParameterError } from '../errors';
 import { InviteLinkModel } from './InviteLinkModel';
 
+export type DbUserDetails = {
+    user_id: number;
+    user_uuid: string;
+    first_name: string;
+    last_name: string;
+    created_at: Date;
+    is_tracking_anonymized: boolean;
+    email: string | undefined;
+    organization_uuid: string;
+    organization_name: string;
+};
 export type DbOrganizationUser = Pick<
     DbUserDetails,
     'user_uuid' | 'first_name' | 'last_name' | 'email'
 >;
 
+export const mapDbUserDetailsToLightdashUser = (
+    user: DbUserDetails,
+): LightdashUser => {
+    if (!user.organization_uuid || !user.organization_name) {
```

Comment:
@ZeRego this logic is because the data model doesn't require the user to be in any organization. However, the app currently assumes that all users live in the one and only project.

I'd like to make this cleaner but the check does the job